cli/cli_test.go

```diff
@@ -149,5 +152,26 @@ var _ = Describe("CLI", func() {
 			Expect(err).To(Succeed())
 			Expect(node.Value).To(Equal("success"))
 		})
+
+		It("Does not change working directory", func() {
+			absoluteConfigPath := path.Join(workingDir(), configPath)
+			wrappedFuncCalled := false
+			expectedWorkingDir := "/"
+			Expect(os.Chdir(expectedWorkingDir)).To(Succeed())
```

Comment:
why this can't be substituted with `expectedWorkingDir "= workingDir()` without all this magic with changing dirs?

Reply:
Good question, I added a comment about this.

Simply asserting that working dir is the same before and after, would not catch the bug when working dir is already the same as the one used by tested function (e.g. when tested function was executed in a previous test case).